src/pages/LoginPage.tsx

```diff
@@ -0,0 +1,101 @@
+import "../components/modal/modal.scss";
+import Email from "../components/Email";
+import Password from "../components/PasswordAndConfirm";
+import {Formik, Field, Form} from "formik";
+import * as Yup from 'yup';
+import Button from "../components/Button";
+import {Link} from "react-router-dom";
+import {useState} from "react";
+import {useAppDispatch} from "../store/hooks";
+import {getLogin} from "../store/loginName/loginSlice"
+
+const minSymbol = 6;
+const maxSymbol = 256;
+const mailRgex = /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/;
+const passwordRegex = new RegExp("^[-/=!#$%&'*+?^_`{|}~.A-Z0-9]{" + minSymbol + "," + maxSymbol + "}$", "i");
+
+interface FormValues {
+  email: string;
+  password: string;
+  remember: boolean;
+}
+
+const SignupSchema = Yup.object().shape({
+  email: Yup.string()
+    .matches(mailRgex)
+    .min(3, 'Email should be more than 3 characters')
```

Comment:
please use constants for such 3 / 256 etc.

https://jeffreyeverhart.com/2020/09/28/javascript-tips-using-named-constants-for-magic-numbers/